Update java compatibility for Ignite and add documentation for cassandra keyspace extraction (#27460)
DataDog/documentation: content/en/tracing/trace_collection/library_config/java.md

**Default**: `true`<br>
When set to `true`, the query string parameters are added to Elasticsearch and OpenSearch spans.

`trace.cassandra.keyspace.statement.extraction.enabled`
: **Environment Variable**: `DD_TRACE_CASSANDRA_KEYSPACE_STATEMENT_EXTRACTION_ENABLED` <br>
**Default**: `false`<br>

> By default, the keyspace is extracted only if configured when creating a session. When set to `true`, the keyspace can also be extracted by looking at metadata in the queries' result.

Reviewer: ```suggestion
By default, the keyspace is extracted only if it is configured during session creation. When set to `true`, the keyspace can also be extracted by examining the metadata in the query results.
```
